docs/index.rst

```diff
@@ -112,8 +112,8 @@ When using tedana, please include the following citations:
       }
       </script>
       <p>
-      <span id="tedana_citation">tedana</span>
-      This link is for the most recent version of the code and that page has links to DOIs
+      <span id="tedana_citation"></span>
+      1. This link is for the most recent version of the code and that page has links to DOIs
       for older versions. To support reproducibility, please cite the version you used:
       <a id="tedana_doi_url" href="https://doi.org/10.5281/zenodo.1250561">https://doi.org/10.5281/zenodo.1250561</a>
```

Comment:
This isn't an issue for this PR, but it looks like there aren't zenodo DOIs for tedana versions past 23.0.1. @tsalo Do you know who/how to add version to zenodo and where in our new release workflow that should be happening?

Reply:
It looks like the Zenodo setting was somehow disabled. I just re-enabled it, so it should work the next time we make a release.